recipes/interlap/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "interlap" %}
+{% set version = "0.2.6" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a8585a165bf7e94d4d262811b4cb00674059414c50afc7561c949b0feb78fa09" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
```

Comment:
Don't use jinja templating here. Just the explicit import.

Reply:
Thanks for the comment! Just for curiosity: Which advantage does the explicit import have?